### PR TITLE
Add enrollment settings commands

### PR DESCRIPTION
**Pull request by oNaiPs**

### CODE REVIEW

File: cmd/root-settings.go [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	Short: "Operations on enrollment",
+}
+
+func init() {
+	rootCmd.AddCommand(settingsCmd)
+
+	settingsCmd.AddCommand(settingsAgentConfigCmd)
+	settingsCmd.AddCommand(settingsEnrollmentCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (ricardo-barracuda) [inline cmd/root-settings.go:46]: Do we need these comments ?

File: cmd/settings-agent-configuration-get.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+		"-",
+		"-",
+		"-",
+	})
+}
+
+func init() {
+	settingsAgentConfigCmd.AddCommand(getAgentConfigCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (ricardo-barracuda) [inline cmd/settings-agent-configuration-get.go:106]: Ditto

File: cmd/settings-agent-configuration-set.go [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+				agentConfigTableWriterAppendError(tw, err, id)
+			})
+		return printListOutputAndError(cmd, createdList, tw, total, err)
+	},
+}
+
+func init() {
+	settingsAgentConfigCmd.AddCommand(setAgentConfigCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (ricardo-barracuda) [inline cmd/settings-agent-configuration-set.go:116]: Ditto

File: cmd/settings-enrollment-get.go [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+	tw.AppendRow(table.Row{
+		"[ERR]",
+		processErrorResponse(err),
+	})
+}
+
+func init() {
+	settingsEnrollmentCmd.AddCommand(getEnrollmentCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (ricardo-barracuda) [inline cmd/settings-enrollment-get.go:92]: Ditto

File: cmd/settings-enrollment-set.go [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+				enrollmentTableWriterAppendError(tw, err, id)
+			})
+		return printListOutputAndError(cmd, createdList, tw, total, err)
+	},
+}
+
+func init() {
+	settingsEnrollmentCmd.AddCommand(setEnrollmentCmd)
+
+	// Here you will define your flags and configuration settings.
```
Comment (ricardo-barracuda) [inline cmd/settings-enrollment-set.go:86]: Ditto

**Review by ricardo-barracuda — APPROVED** [review]

File: cmd/root-settings.go (hunk 1 above)
Comment (oNaiPs) [reply cmd/root-settings.go:46]: All the other root files have it, i'll leave it for now for consistency, but agree on removing in all files

**Review by pablosole — APPROVED** [review]

**Pull request merged**